[2022-11-29 12:44:25,533] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: Covid_dags.creatableLoad manual__2022-11-29T12:39:19.579272+00:00 [queued]>
[2022-11-29 12:44:25,545] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: Covid_dags.creatableLoad manual__2022-11-29T12:39:19.579272+00:00 [queued]>
[2022-11-29 12:44:25,545] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-11-29 12:44:25,545] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-11-29 12:44:25,546] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-11-29 12:44:25,558] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): creatableLoad> on 2022-11-29 12:39:19.579272+00:00
[2022-11-29 12:44:25,564] {standard_task_runner.py:52} INFO - Started process 171 to run task
[2022-11-29 12:44:25,568] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Covid_dags', 'creatableLoad', 'manual__2022-11-29T12:39:19.579272+00:00', '--job-id', '1114', '--raw', '--subdir', 'DAGS_FOLDER/covid.py', '--cfg-path', '/tmp/tmpggdemnam', '--error-file', '/tmp/tmplo3d7xia']
[2022-11-29 12:44:25,569] {standard_task_runner.py:77} INFO - Job 1114: Subtask creatableLoad
[2022-11-29 12:44:25,633] {logging_mixin.py:109} INFO - Running <TaskInstance: Covid_dags.creatableLoad manual__2022-11-29T12:39:19.579272+00:00 [running]> on host acb48b0a6e0d
[2022-11-29 12:44:25,701] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Covid_dags
AIRFLOW_CTX_TASK_ID=creatableLoad
AIRFLOW_CTX_EXECUTION_DATE=2022-11-29T12:39:19.579272+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-29T12:39:19.579272+00:00
[2022-11-29 12:44:25,929] {python.py:175} INFO - Done. Returned value was: None
[2022-11-29 12:44:25,942] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=Covid_dags, task_id=creatableLoad, execution_date=20221129T123919, start_date=20221129T124425, end_date=20221129T124425
[2022-11-29 12:44:25,982] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-11-29 12:44:26,017] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
